### PR TITLE
Disable VMR updates for `dotnet/scenario-tests`

### PR DESCRIPTION
**Pull request by premun**
https://github.com/dotnet/arcade-services/issues/4159

### CODE REVIEW

**Review by dkurepa — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Note that scenario-tests official build is currently broken because of a failing test. cc @mmitche

**Review by ViktorHofer — APPROVED** [review]

**Comment by premun** [comment]
I will wait with merging this until we get a PR opened against VMR successfully, so let scenario-tests flow through SDK if you see/need it please

**Review by mthalman — APPROVED** [review]

File: src/VirtualMonoRepo/source-mappings.json [hunk 1]
```diff
@@ -112,7 +112,8 @@
         },
         {
             "name": "scenario-tests",
-            "defaultRemote": "https://github.com/dotnet/scenario-tests"
+            "defaultRemote": "https://github.com/dotnet/scenario-tests",
+            "disableSynchronization": true
```
Comment (MichaelSimons) [inline src/VirtualMonoRepo/source-mappings.json:116]: Is this a point in time setting that will eventually go away once the repo darc flows are all defined?

Comment (premun) [reply src/VirtualMonoRepo/source-mappings.json:116]: Yes, this whole file will go away most likely

**Pull request merged**